src/components/Button/Button.styled.ts

```diff
@@ -198,6 +198,22 @@ const BlueOnWhite = styled(ButtonWrapper)`
   }
 `;
 
+const WhiteOnBlack = styled(ButtonWrapper)`
+  & > button {
+    color: ${theme.colors.white};
+    border-color: ${theme.colors.white};
+  }
+
+  & > button:hover {
+    background: ${theme.colors.white}12;

```

Comment:
I was having a lot of trouble seeing the opacity when hovering, so I decided to use an accessibility tool and I'd suggest changing it to something between 30 and 50 
![image](https://user-images.githubusercontent.com/23409284/169141108-c4466fd8-7e1f-4b57-b8cf-e2ea32d9a704.png)